Thrive.csproj

```diff
@@ -291,4 +292,4 @@
     <Content Include="simulation_parameters\microbe_stage\organelles.json" />
     <Content Include="simulation_parameters\microbe_stage\species_names.json" />
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
What's going on with this?

Reply:
Probably changed the fact that the file ends / doesn't end with a newline. CI should complain about this.